transfer/transfer-02-file-transfer-listener/README.md

```diff
@@ -52,6 +52,13 @@ java -Dedc.fs.config=transfer/transfer-02-file-transfer-listener/file-transfer-l
 ./gradlew transfer:transfer-01-file-transfer:file-transfer-provider:build
 java -Dedc.fs.config=transfer/transfer-01-file-transfer/file-transfer-provider/config.properties -jar transfer/transfer-01-file-transfer/file-transfer-provider/build/libs/provider.jar
 ````
+```Windows command prompt
```

Comment:
spacing

Reply:
I push the changes for spacing, capitalization. Fow duplication issue, i make an extra comment to safe user from doing unwanted mistake like , (not follow the 'Mac user' commands which have no change for 'windows user' ).